### PR TITLE
Feature/translation pt

### PR DESCRIPTION
**Pull request by luispinho**
**WIP** (still more translations to be done, other PRs will be opened in the future).
 
Complete Portuguese translations for Home Screen.
Add other various Portuguese translations.

### CODE REVIEW

File: apps/client/src/locales/messages.pt.xlf [hunk 1]
```diff
@@ -3243,15 +3243,15 @@
...
+        <target state="new">Risco mais Elevado</target>
         <context-group purpose="location">
           <context context-type="sourcefile">libs/ui/src/lib/i18n.ts</context>
           <context context-type="linenumber">13</context>
         </context-group>
       </trans-unit>
       <trans-unit id="4152514811781104574" datatype="html">
         <source>Lower Risk</source>
-        <target state="new">Lower Risk</target>
+        <target state="new">Risco menos Elevado</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.pt.xlf:3254]: state="translated"

File: apps/client/src/locales/messages.pt.xlf [hunk 2]
```diff
@@ -3243,15 +3243,15 @@
       </trans-unit>
       <trans-unit id="2963674907100579427" datatype="html">
         <source>Higher Risk</source>
-        <target state="new">Higher Risk</target>
+        <target state="new">Risco mais Elevado</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.pt.xlf:3246]: state="translated"

File: apps/client/src/locales/messages.pt.xlf [hunk 3]
```diff
@@ -1955,31 +1955,31 @@
...
+        <target state="translated">Atualizar conta</target>
         <context-group purpose="location">
           <context context-type="sourcefile">apps/client/src/app/pages/accounts/create-or-update-account-dialog/create-or-update-account-dialog.html</context>
           <context context-type="linenumber">2</context>
         </context-group>
       </trans-unit>
       <trans-unit id="4eb2164d8ef5d328ef6bc1e66da011218bac968f" datatype="html">
         <source>Add account</source>
-        <target state="new">Add account</target>
+        <target state="new">Adicionar conta</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.pt.xlf:1982]: state="translated"

**Review by dtslvr — CHANGES_REQUESTED** [review]
Looks fine. Just found some missing "new" -> "translated" 🙂

**Comment by luispinho** [comment]
> Looks fine. Just found some missing "new" -> "translated" 🙂

Fixed 👍

**Review by dtslvr — APPROVED** [review]

**Comment by dtslvr** [comment]
> Fixed 👍

Thank you @luispinho

**Pull request merged**